core-java-modules/core-java-21/src/main/java/com/baeldung/java21/StringTemplates.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.java21;
+
+public class StringTemplates {
+
+    public String getStringTemplate() {
+        String name = "Baeldung"; 
+        String welcomeText = STR."Welcome to \{name}"; 
+        return welcomeText;
```

Comment:
We can simplify to:
```suggestion
        return STR."Welcome to \{name}";
```

Anyway, this is a feature in preview, isn't it?

Reply:
Yes, it is a preview feature. I have simplified the return statement.